### ISSUE TITLE
Create setup.py

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/beprof#1]
Make it possible to install this is as pip package

**Comment by matkrak** [comment]
Shall I reorganize files in the repository so that one could install it as
~$ pip3 install <https_to_repo>,
or upload this package to PyPi (what I already done) so that it can be installed with
~$ pip3 install beprof
Or both maybe?

**Comment by grzanka** [comment]
To early to have it in PyPi, its not mature enough. We should use direct installation from GIT now.

**Comment by matkrak** [comment]
I have made changes in setup.py just as you wished. Anything else that should be done before this issue is closed?
